app/server/routes/helpCentreFrontend.ts

```diff
@@ -1,4 +1,5 @@
 import { Request, Response, Router } from "express";
+import { defaultPageTitle } from "../../client/components/helpCentre/helpCentrePageTitle";
```

Comment:
Hi Kelvin,

In this project we avoid having the server-side code import anything from the client-side code and vice-versa. When both client and server sides need a particular variable or function we put it in the shared folder.

Reply:
That makes sense.  Presumably this isn't even accessible to the serverside